Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -5444,6 +5558,26 @@
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
+		3A15783C28D1505B00142DB1 /* TestWGSL */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 3A15785028D1505B00142DB1 /* Build configuration list for PBXNativeTarget "TestWGSL" */;
+			buildPhases = (
+				3A15783D28D1505B00142DB1 /* Product Dependencies */,
+				3A15783F28D1505B00142DB1 /* Sources */,
+				3A15784628D1505B00142DB1 /* Frameworks */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				3A5DDAEC28D156BB004DA950 /* PBXTargetDependency */,
```

Comment:
Is this target dependency on WGSL necessary? Having it means that any attempt to build TestWebKitAPI will also build WGSL, including `make` builds and production builds, which could be a problem. Can we remove it?

Because TestWGSL links against libwgsl.a (on L3496), there is already an implicit dependency. So any scheme with "Find Implicit Dependencies" turned on will build WGSL and TestWGSL in the correct order.